classes/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './item'
```

Comment:
- [ ] Pllease change this route from its origin
```suggestion
require_relative '../item'
```